lib/ui/semantics/attributed_string_unittests.cc

```diff
@@ -0,0 +1,122 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef DEBUG
+#define DEBUG
+#endif
```

Comment:
Why do we need this?

Reply:
oops I forgot to remove this, I added list to use util function in dartsdk, which i eventually gave up.